Apps/Examples/Examples/All Examples/Custom2DPuckExample.swift

```diff
@@ -50,37 +142,72 @@ public class Custom2DPuckExample: UIViewController, ExampleProtocol {
         finish()
     }
 
-    @objc func showHideAccuracyRadius() {
-        showsAccuracyRing.toggle()
+    private func customizePuckButton() {
+        // Set up button to change the puck options
+        let button = UIButton(type: .system)
+        button.setTitle("Customize puck", for: .normal)
+        button.setTitleColor(.white, for: .normal)
+        button.backgroundColor = #colorLiteral(red: 0, green: 0.4784313725, blue: 0.9882352941, alpha: 1)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.layer.cornerRadius = 20
+        button.addTarget(self, action: #selector(changePuckOptions(sender:)), for: .touchUpInside)
+        view.addSubview(button)
+
+        // Set button location
+        let horizontalConstraint = button.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -24)
+        let verticalConstraint = button.centerXAnchor.constraint(equalTo: view.centerXAnchor)
+        let widthConstraint = button.widthAnchor.constraint(equalToConstant: 200)
+        let heightConstraint = button.heightAnchor.constraint(equalToConstant: 40)
+        NSLayoutConstraint.activate([horizontalConstraint, verticalConstraint, widthConstraint, heightConstraint])
```

Comment:
Nit: There is no need to make a variables if you are not going to change them later
```suggestion
        NSLayoutConstraint.activate([
            button.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -24),
            button.centerXAnchor.constraint(equalTo: view.centerXAnchor),
            button.widthAnchor.constraint(equalToConstant: 200),
            button.heightAnchor.constraint(equalToConstant: 40)
        ])
```